controllers/articles.controller.js

```diff
@@ -0,0 +1,13 @@
+const retrieveArticleById = require("../models/articles.model");
+
+function getArticleById(request, response, next) {
+  const articleId = request.params.article_id;
+
+  retrieveArticleById(articleId)
+    .then((article) => {
+      response.status(200).send(article);
```

Comment:
Send over an object here so you can hold the article on a key, this will help a lot when we get to front end. 

send({article: article}) or just ({article})

If you do this, you will have to just readjust your test to look on the key of article